spec/requests/publishers/job_applications_spec.rb

```diff
@@ -0,0 +1,162 @@
+require "rails_helper"
+
+RSpec.describe "Job applications", type: :request do
+  let(:organisation) { build(:school) }
+  let(:vacancy) { create(:vacancy, organisation_vacancies_attributes: [{ organisation: organisation }]) }
+  let(:job_application) { create(:job_application, :status_submitted, vacancy: vacancy) }
+  let(:publisher) { create(:publisher, accepted_terms_at: 1.day.ago) }
+
+  before do
+    allow(JobseekerApplicationsFeature).to receive(:enabled?).and_return(true)
+    # TODO: Remove this stub when publishers are associated to organisations
```

Comment:
this can be done now

Reply:
We actually still need to do this. Not sure whether to leave this or stub the session instead though?